plugin/fzf.vim

```diff
@@ -405,6 +405,10 @@ function! s:cmd_callback(lines) abort
   endif
   let key = remove(a:lines, 0)
   let cmd = get(s:action, key, 'e')
+  augroup fzf_swap
+    autocmd SwapExists * let v:swapchoice='o'
+          \| call s:warn('fzf: E325: swap file exists: '.expand('<afile>'))
```

Comment:
This does not appear in the statusline, but it will be helpful in case user wonders why a file was opened RO.